recipes/vitap/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "vitap" %}
+{% set version = "1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/DrKaiyangZheng/VITAP/archive/refs/heads/main.zip
```

Comment:
I recommend you to make a release/tag and use it instead of `main`.
Now if you push a commit to `main` the next build will fail due to different SHA256.

Reply:
I have made a release/tag and use it instead of main. Thanks.